rclip/main.py

```diff
@@ -71,7 +83,10 @@ def _index_files(self, filepaths: List[str], metas: List[ImageMeta]):
         images.append(image)
         filtered_paths.append(path)
       except PIL.UnidentifiedImageError as ex:
-        pass
+        image = self._read_raw_file(path)
+        if not image:
+          images.append(image)
+          filtered_paths.append(path)
```

Comment:
I wonder if we can check the image file type explicitly to choose whether to use `PIL` or `rawpy` instead of using an exception for this. What do you think?

Reply:
It's implemented as suggested. I have created 2 different methods and called those according to the extensions.